experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -593,3 +587,87 @@ if isfile(
         ),
     )
 end
+
+# Water content in soil and snow
```

Comment:
Added in some plots that were accidentally removed at one point, and added SWE to it